pkg/storage-init/storage-init.sh

```diff
@@ -3,6 +3,88 @@
 # Copyright (c) 2018 Zededa, Inc.
 # SPDX-License-Identifier: Apache-2.0
 
+zfs_set_parameter() {
+    parameter="$1"
+    value="$2"
+
+    echo "${value}" > /sys/module/zfs/parameters/"${parameter}"
+}
+
+zfs_set_arc_limits() {
+    # NOOP if persist is not ZFS
+    if ! read -r persist_fs < /run/eve.persist_type; then
+       echo "Can not determine persist filesystem type"
+       return 1
+    fi
+
+    if [ "${persist_fs}" != "zfs" ]; then
+       return 0
+    fi
+
+    # Constants
+    zfs_arc_min="$(echo "256*1024*1024" | bc)"
```

Comment:
FWIW no need to invoke bc for this; shell can do the arithmetic just fine.

Reply:
addressed in the new version of PR